benchmarks/src/main/java/io/druid/benchmark/ColumnScanBenchmark.java

```diff
@@ -0,0 +1,292 @@
+package io.druid.benchmark;
```

Comment:
@fjy Added header

I can add an initial benchmarks doc to content/development, that sounds useful